contracts/minting/MintController.sol

```diff
@@ -100,6 +101,23 @@ contract MintController is Controller {
         return internal_setMinterAllowance(minter, newAllowance);
     }
 
+    /**
+     * @dev decreases the minter allowance if and only if the minter is
+     * currently active. The controller can safely send a signed decrementMinterAllowance()
+     * transaction to a minter and not worry about it being used to undo a removeMinter()
+     * transaction.
+     */
+    function decrementMinterAllowance(uint256 allowanceDecrement) onlyController public returns (bool) {
```

Comment:
please use solidity convention for input `_allowanceDecrement`

Reply:
done